src/hooks/translations.tsx

```diff
@@ -0,0 +1,36 @@
+import { useQuery } from "@apollo/react-hooks";
+import { GET_TRANSLATIONS } from "$queries";
+import { ApolloError } from "apollo-client";
+
+interface ITranslation {
+  key: string;
+  value: string;
+}
+
+type HookedFuncResult<T> = {
+  translations: T | null;
+  error: ApolloError | undefined;
+  loading: boolean;
+};
+
+const translationMapper = (translationsList: ITranslation[]) => {
+  const translations: any = {};
+  for (const {key, value} of translationsList) {
```

Comment:
Aca tmb dejaria un espacio entre llaves ` for (const { key, value } of translationsList) {`

Reply:
https://palantir.github.io/tslint/rules/whitespace/

habria que tirar un pr que integre esta regla, para no tener que verla a ojo

Reply:
Queres que tire pr de eso? habria que probarlo con el back y el front